python/cmsis_svd/parser.py

```diff
@@ -66,8 +66,8 @@ class SVDParser(object):
     expand_arrays_of_registers = 0
 
     @classmethod
-    def for_xml_file(cls, path):
-        return cls(ET.parse(path))
+    def for_xml_file(cls, path, remove_reserved = 0, expand_arrays_of_registers = 0):
```

Comment:
Really minor, kwargs should have no space around the `=` according to PEP8.   E.g. `remove_reserved=0`.